connectioncmd/connection.go

```diff
@@ -0,0 +1,73 @@
+package connectioncmd
+
+import (
+	"github.com/charmbracelet/bubbles/list"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/hazelcast/hazelcast-commandline-client/config"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/file"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func New() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "connection",
```

Comment:
IMO, the name of the command "connection" and the descriptions do not convey the functionality of this command. Do you have other options in your mind?

Maybe such as:
CmdName: "connection-wizard"
Desc.: "Assist with connection configuration"

This is just an example. I'm sure you have better things on mind 🙂

Reply:
`connection-wizard` sounds good, refactored at https://github.com/hazelcast/hazelcast-commandline-client/pull/139/commits/8a912812f9624431cb43dd10992a1e2135072552.